impeller/renderer/backend/gles/buffer_bindings_gles.cc

```diff
@@ -219,8 +219,8 @@ bool BufferBindingsGLES::BindVertexAttributes(const ProcTableGLES& gl,
 
 bool BufferBindingsGLES::BindUniformData(
     const ProcTableGLES& gl,
-    const std::vector<TextureAndSampler>& bound_textures,
-    const std::vector<BufferResource>& bound_buffers,
+    const TextureAndSampler bound_textures[],
+    const BufferResource bound_buffers[],
```

Comment:
Why are you converting these to c arrays?  It looks like everything going into this is a vector anways.

Reply:
I'm using a fixed size std::array in render_pass_gles3.

Reply:
Missed that, thanks.  We'll be able to use std::span in c++20.

Reply:
god I'm so ready for span